azure/azure-external-release-pipeline.yml

```diff
@@ -0,0 +1,48 @@
+name: "$(SourceBranchName)+$(BuildID)"
+
+trigger: none
+pr: none
+
+resources:
+  repositories:
+    - repository: common
+      type: github
+      name: NHSDigital/api-management-utils
+      endpoint: NHSDigital
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+variables:
+  - template: project.yml
+  - group: pact
+  - group: variable-group-read
+  - group: eps-ref-smoke-tests
+
+extends:
+  template: azure-release-template.yml
+  parameters:
+    service_name: ${{ variables.service_name }}
+    short_service_name: ${{ variables.short_service_name }}
+    service_base_path: ${{ variables.service_base_path }}
+    product_display_name: ${{ variables.product_display_name }}
+    product_description: ${{ variables.product_description }}
+    service_base_path_pr: none
+    spec_file: ${{ variables.spec_file }}
+    apigee_deployments:
+      - environment: sandbox
+        proxy_path: sandbox
+        depends_on:
```

Comment:
Do these need updating to depends_on: [] now that internal_* environments are in the internal pipeline? If not specified then these get depends_on defaults from common pipeline, [] overrides

Reply:
Yeah possibly. Trying the new pipelines out in Azure DevOps now